clickhouse-admin/api/src/lib.rs

```diff
@@ -95,3 +77,28 @@ pub trait ClickhouseAdminApi {
         rqctx: RequestContext<Self::Context>,
     ) -> Result<HttpResponseOk<ClickhouseKeeperClusterMembership>, HttpError>;
 }
+
+/// We separate the admin interface for the keeper and server APIs because they
```

Comment:
Here as well

Reply:
Done in [0a2e28b](https://github.com/oxidecomputer/omicron/pull/6837/commits/0a2e28b22169bf55587164395a620fb00bef3d4c)